src/ir_Vestel.h

```diff
@@ -0,0 +1,139 @@
+// Copyright 2018 Erdem U. Altinyurt
+// (Used Midea.h as template)
+#ifndef IR_VESTEL_H_
+#define IR_VESTEL_H_
+
+#define __STDC_LIMIT_MACROS
+#include <stdint.h>
+#ifdef ARDUINO
+#include <Arduino.h>
+#else
+#include <string>
+#endif
+#include "IRremoteESP8266.h"
+#include "IRsend.h"
+
+//                 VV     VV  EEEEEEE   SSSSS  TTTTTTTT  EEEEEEE  LL
+//                 VV     VV  EE       S          TT     EE       LL
+//                  VV   VV   EEEEE     SSSS      TT     EEEEE    LL
+//                   VV VV    EE            S     TT     EE       LL
+//                    VVV     EEEEEEE  SSSSS      TT     EEEEEEE  LLLLLLL
+
+// Vestel added by Erdem U. Altinyurt
+
+// Constants
+const uint16_t kVestelACHdrMark = 3110;
+const uint16_t kVestelACHdrSpace = 9066;
+const uint16_t kVestelACBitMark = 520;
+const uint16_t kVestelACOneSpace = 1535;
+const uint16_t kVestelACZeroSpace = 480;
+const uint16_t kVestelACTolerance = 30;
+
+const uint8_t kVestelACMinTempH = 16;
+const uint8_t kVestelACMinTempC = 18;
+const uint8_t kVestelACMaxTemp  = 30;
+
+const uint64_t kVestelACCRCMask = 0xFFFFFFFFFFF00000;
+
+const uint8_t kVestelACAuto = 0;
+const uint8_t kVestelACCool = 1;
+const uint8_t kVestelACDry  = 2;
+const uint8_t kVestelACFan  = 3;
+const uint8_t kVestelACHeat = 4;
+
+const uint8_t kVestelACFanAuto = 1;
+const uint8_t kVestelACFanLow  = 5;
+const uint8_t kVestelACFanMed  = 9;
+const uint8_t kVestelACFanHigh = 0xB;
+
+const uint8_t kVestelACNormal  = 1;
+const uint8_t kVestelACSleep   = 3;
+const uint8_t kVestelACTurbo   = 7;
+const uint8_t kVestelACIon     = 4;
+const uint8_t kVestelACWing    = 0xA;
+
+union VestelACState{
```

Comment:
We are getting the compiler _warnings_ when this compiles:
```
../src/ir_Vestel.h:57:9: note: offset of packed bit-field ‘VestelACState::<anonymous struct>::CRC’ has changed in GCC 4.4
   struct{
         ^
../src/ir_Vestel.h:57:9: note: offset of packed bit-field ‘VestelACState::<anonymous struct>::zero’ has changed in GCC 4.4
../src/ir_Vestel.h:76:9: note: offset of packed bit-field ‘VestelACState::<anonymous struct>::t_CRC’ has changed in GCC 4.4
   struct{
         ^
../src/ir_Vestel.h:76:9: note: offset of packed bit-field ‘VestelACState::<anonymous struct>::t_turnOffHour’ has changed in GCC 4.4
../src/ir_Vestel.h:76:9: note: offset of packed bit-field ‘VestelACState::<anonymous struct>::t_turnOnHour’ has changed in GCC 4.4
../src/ir_Vestel.h:76:9: note: offset of packed bit-field ‘VestelACState::<anonymous struct>::t_hour’ has changed in GCC 4.4
```
I **think** this may address them. Untested/not sure.

```cpp
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wpacked-bitfield-compat"
union VestelACState{
...
};
#pragma GCC diagnostic pop
```

Ref:
  https://stackoverflow.com/questions/41253759/ignoring-note-offset-of-packed-bit-field-without-using-wno-packed-bitfield-co
  https://www.gnu.org/software/gcc/gcc-4.4/changes.html

Reply:
Well, not warnings but _notes_ this time. :)

I add those pragma messages but doesn't kill the notes at Arduino GCC4.4